waterlog/src/Components/AlertTable.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import './../Stylesheets/_alerts.scss';
+import { LowStatusIcon, MediumStatusIcon, HighStatusIcon } from './AlertBox';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchSegmentsLeaks } from '../actions/SegmentLeaksActions';
+
+class AlertTableComponent extends Component {
+	componentWillMount() {
+		this.props.fetchSegmentsLeaks();
+	}
+	alertInfo = [ 'DATE', 'DESCRIPTION', 'COST', 'STATUS' ];
+
+	render() {
+		
+		return (
+			<table>
```

Comment:
As an upcoming suggestion, use table for tabular datas.
When it comes to the presentation of your page, make use of div, aside, section, nav and many other elements coming with HTML5.

Reply:
Will take into consideration.